dirac.cfg

```diff
@@ -209,6 +209,47 @@ Systems
          ReqProxyURLs = dips://server1:9161/RequestManagement/ReqProxy, dips://server2:9161/RequestManagement/ReqProxy
       }
    }
+   TransformationSystem
+   {
+      Agents
+      {
+          TransformationCleaningAgent
```

Comment:
The `literalinclude` does not work here?

Reply:
It also works with the dirac.cfg file, but the proximity of the ConfigTemplate to the code, and the fact that the ConfigTemplate needs to be there makes me kind of prefer using the ConfigTemplate.cfg
Also as there is so much more content in dirac.cfg I was afraid of accidental matching, though with @chaen's ##BEGIN ##END tags this wouldn't happen.
So then the only problem is the question of duplication of information.

Reply:
Then the question is more whether we want to put these tags in the configTemplate or in the dirac.cfg
Forsake of dirac.cfg readability, I'd rather put it in the configTemplate

Reply:
Shall we discuss this at the next BiLD? We now all got "excited" on using `literainclude`, which, we all see it, may mean that we can avoid cfg duplications, as long as we properly adapt the configTemplate files. If done correctly this can certainly be beneficial, but we also need to consider that these configTemplate files are used to host the initial configuration for installing agents and services, so everything we put there would end up in the CS the first time that the agent is installed (e.g. via the dirac-admin-sysadmin-cli).